solidity/package.json

```diff
@@ -45,7 +45,7 @@
     "ethereumjs-abi": "^0.6.5",
     "mocha": "^7.1.1",
     "mocha-junit-reporter": "^1.23.1",
-    "solc": "^0.5.7",
+    "solc": "^0.5.17",
```

Comment:
Don't we want it to be `"solc": "0.5.17"`?

Reply:
Yeah probably, though incidentally it doesn't matter because we explicitly set `solc` in `truffle-config.json`.